src/js/utils.js

```diff
@@ -58,10 +58,11 @@ export function iconPad(props) {
   let style = '';
   // browser default is 16px, but accommodate if app has modified
   // include fallback in case window is undefined
-  const rootFontSize = parseMetricToNum(
-    window?.getComputedStyle(document.body).getPropertyValue('font-size') ||
-      '16px',
-  );
+  if (typeof window !== 'undefined') {
+    rootFontSize = parseMetricToNum(
+      window.getComputedStyle(document.body).getPropertyValue('font-size') || '16px'
```

Comment:
```suggestion
      window?.getComputedStyle(document.body).getPropertyValue('font-size') || '16px'
```
Just to mirror exactly what's in Grommet, which has been working, should we do this? ^

Although I think you "if statement" above should check this anyway.

Reply:
Also I wonder if if should actually be like this:

```
let rootFontSize = '16px';
if (typeof window !== undefined) {
rootFontSize = parseMetricToNum(
      window?.getComputedStyle(document.body).getPropertyValue('font-size')
}
```